changelogs/fragments/880-Functional_tso_command_test_cases_currently_can_not_be_run_concurrently.yml

```diff
@@ -0,0 +1,4 @@
+bugfix:
```

Comment:
i'd recategorize this to 'trivial' which will skip including this fragment in the changelog since these changes are to the tests and not to the module/module_utils, from a user pespective, there's no change.